src/components/rating/rating.tsx

```diff
@@ -222,11 +221,48 @@ export class Rating implements LabelableComponent, FormComponent, InteractiveCom
     this.setFocus();
   }
 
+  /**
+   * update `value` property of the rating component
+   *
+   * @param value value used to update `value` property
+   * @internal
+   */
   private updateValue(value: number) {
     this.value = value;
     this.calciteRatingChange.emit({ value });
   }
 
+  /**
+   * register keyboard input and reset `value` property to `0` if user enters 'Space' or 'Enter'
+   *
+   * @param event keyboard event
+   * @internal
```

Comment:
I don't think these private methods need to be declared as `@internal` or documented at all